qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -213,11 +213,14 @@ def _submit_job(
                 job_share_level=job_share_level,
                 job_tags=job_tags)
         except ApiError as ex:
+            logger.debug('Error: There was an error submitting the job: %s.', str(ex))
```

Comment:
I don't really see the value of logging the error when it's already being raised...